modules/odc-using-the-devfile-registry.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: PROCEDURE
+[id="odc-using-the-devfile-registry_{context}"]
+= Using the Devfile Registry to access product Devfiles
+
+You can use the devfiles in the *+Add* flow of the *Developer* perspective to create an application. The *+Add* flow provides a complete integration with the https://registry.devfile.io/viewer[public community registry]. A devfile is a portable YAML file that describes your development environment without needing to configure it from scratch. Using the Devfile registry, you can use a pre-configured devfile to create an application.
+
+.Procedure
+
+. Click *Developer Perspective* > *+Add* > *Developer Catalog* > *All Services* to view all the available services in the Developer Catalog.
```

Comment:
This should be Navigate to...*All Services*. A list of all the available...Catalog is displayed.
All the arrows should be formatted as shown in the table here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference 
Developer Catalog needs to be in bold.